include/tvm/relax/expr.h

```diff
@@ -983,6 +983,8 @@ constexpr const char* kCodegen = "Codegen";
 constexpr const char* kComposite = "Composite";
 /*! \brief Indicate the function was created by the Pattern Partitioning Pass. */
 constexpr const char* kPartitionedFromPattern = "PartitionedFromPattern";
+/*! \brief Indicate the function cannot be lifted to transform_params*/
+constexpr const char* kStopLifting = "StopLifting";
```

Comment:
can we instead add a special relax op, `relax.builtin.stop_lifting`, which is an identity op, and can be used to mark the boundary